docs/terminal/Cairns TCU/coordination.md

```diff
@@ -0,0 +1,104 @@
+---
+  title: Coordination
```

Comment:
Page is a copy of ML TCU coordination

Reply:
Yeah it's just a place holder, haven't done the actual page, just putting the TCU diagrams there

Reply:
Ah all good